presto-hive/src/main/java/com/facebook/presto/hive/OrcFileWriterFactory.java

```diff
@@ -64,31 +67,39 @@
     private final HdfsEnvironment hdfsEnvironment;
     private final TypeManager typeManager;
     private final NodeVersion nodeVersion;
-    private final FileFormatDataSourceStats stats;
+    private final FileFormatDataSourceStats readStats;
+    private final OrcWriterStats stats = new OrcWriterStats();
 
     @Inject
     public OrcFileWriterFactory(
             HdfsEnvironment hdfsEnvironment,
             TypeManager typeManager,
             NodeVersion nodeVersion,
             HiveClientConfig hiveClientConfig,
-            FileFormatDataSourceStats stats)
+            FileFormatDataSourceStats readStats)
     {
-        this(hdfsEnvironment, typeManager, nodeVersion, requireNonNull(hiveClientConfig, "hiveClientConfig is null").getDateTimeZone(), stats);
+        this(hdfsEnvironment, typeManager, nodeVersion, requireNonNull(hiveClientConfig, "hiveClientConfig is null").getDateTimeZone(), readStats);
     }
 
     public OrcFileWriterFactory(
             HdfsEnvironment hdfsEnvironment,
             TypeManager typeManager,
             NodeVersion nodeVersion,
             DateTimeZone hiveStorageTimeZone,
-            FileFormatDataSourceStats stats)
+            FileFormatDataSourceStats readStats)
     {
         this.hdfsEnvironment = requireNonNull(hdfsEnvironment, "hdfsEnvironment is null");
         this.typeManager = requireNonNull(typeManager, "typeManager is null");
         this.nodeVersion = requireNonNull(nodeVersion, "nodeVersion is null");
         this.hiveStorageTimeZone = requireNonNull(hiveStorageTimeZone, "hiveStorageTimeZone is null");
-        this.stats = requireNonNull(stats, "stats is null");
+        this.readStats = requireNonNull(readStats, "stats is null");
+    }
+
+    @Managed
+    @Flatten
+    public DateTimeZone getHiveStorageTimeZone()
```

Comment:
* (Before I realize this is likely a typo) Out of curiosity, where is JMX serialization for `DateTimeZone` defined?
* (Now that I realized this is likely a typo) Is this supposed to be `getStats` instead? I would run Presto manually with this and verify that the numbers looks reasonable with visualvm.